testing/linter.nu

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env nu
+ls modules/
```

Comment:
I think that all `*.nu` files should be lintered. The following pipeline should do until we have no dir inside a dir:

```suggestion
ls
|append (ls | where type == dir |get name| each {|it| ls $it})
|get name
```

Reply:
Linter now covers all .nu files (but modules are still linted separately from standalone scripts)